lib/galaxy/jobs/__init__.py

```diff
@@ -1181,7 +1181,9 @@ def galaxy_url(self):
         return self.get_destination_configuration("galaxy_infrastructure_url")
 
     def get_job(self) -> model.Job:
-        return self.sa_session.get(Job, self.job_id)
+        job_id = self.sa_session.get(Job, self.job_id)
+        assert job_id
+        return job_id
```

Comment:
Just realised that this is actually a `job` not a `job_id`, so wrong variable name, sorry.

Reply:
I'll squash all the commits from the previous SA20 PR into one to avoid confusion (this one has to build on that one, which is not yet merged)